src/is-plain-object.ts

```diff
@@ -0,0 +1,17 @@
+export function isPlainObject(value: unknown): value is Object {
```

Comment:
if this is copied 1:1 from lodash we should give credit and add a license remark

Reply:
Nope. It is my simplified and somehow optimized version of isPlainObject.